forte/data/ontology/top.py

```diff
@@ -1208,15 +1247,17 @@ class BoundingBox(Box):
 
     """
 
+    _grid_id: int
+
     def __init__(
         self,
         pack: PackType,
-        height: int,
-        width: int,
-        grid_height: int,
-        grid_width: int,
-        grid_cell_h_idx: int,
-        grid_cell_w_idx: int,
+        height: int = 1,
```

Comment:
Why only `grid` move to the class level? Note that it is not just about moving things to data store, but need to have a consistent approach too.

Reply:
I believe that was the original implementation of `BoundingBox`. Do you feel that we should not create the `Grid` object?

Reply:
No, I mean you moved `_grid_id` to be a class variable. Do you understand how these class variable affect the attribute list?

I don't know what you mean by creeating the `Grid` object, but that should not be a problem for this PR.

Reply:
As you suggested, I am using the new implementation of CV ontologies (from @hepengfe 's branch) and making my changes on that so that merging is easier. I believe he has handled the Grid implementation there.

Reply:
ok, probably this PR is a bit intertwined with the `Grid` which makes me hard to judge the correctness right now.